.github/workflows/odh-build-and-publish-operator-image.yaml

```diff
@@ -0,0 +1,102 @@
+# This is a copy of the publish-core-images.yaml and has been customized to
+# use the quay login credentials.
+# The unused parts of the original have been commented out on purpose.
+name: ODH
+
+on:
+  - push
+  - pull_request
+
+jobs:
+    build-and-publish-opeator:
+      name: Build and (or) Publish Image
+      runs-on: ubuntu-latest  
+      env:
+        GOPATH: ${{ github.workspace }}/go
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          
+        - name: Set up Go
+          uses: actions/setup-go@v3
+          with:
+            go-version-file: go.mod
+
+        - name: Run go mod
+          shell: bash
+          run: |
+            go mod download
+                
+        # Build operators inside the gh runner vm directly and then copy the go binaries to docker images using the Dockerfile.buildx
+        - name: Build linux/amd64 operator binary
+          env:
+            CGO_ENABLED: 1
+            GOOS: linux
+            GOARCH: amd64
+          shell: bash
+          run: |
+            go build -tags strictfipsruntime -a -o manager-$GOARCH cmd/training-operator.v1/main.go
+   
+        - name: Build linux/arm64 operator binary
+          env:
+            CC: aarch64-linux-gnu-gcc
+            CGO_ENABLED: 1
+            GOOS: linux
+            GOARCH: arm64
+          shell: bash
+          run: |
+            sudo apt-get update
+            sudo apt-get install -y gcc-aarch64-linux-gnu libc6-dev-arm64-cross
+            go build -tags strictfipsruntime -a -o manager-$GOARCH cmd/training-operator.v1/main.go
+
+        - name: Add docker tags
+          id: meta
+          uses: docker/metadata-action@v5
+          with:
+            images: quay.io/${{ vars.QUAY_REPO_NAME }}/training-operator
+            tags: |
+              type=raw,latest
+              type=sha,prefix=v1-  
```

Comment:
Is it needed?

Reply:
I would argue yes, we can use it to generate the tags to be used in the push. https://github.com/docker/metadata-action?tab=readme-ov-file#semver

Reply:
I thought docker/metadata-action would only work with Docker / Buildx. Does that work with Buildah / Podman?

Reply:
I think is supported according to this: https://github.com/redhat-actions/buildah-build/blob/main/.github/workflows/docker_metadata_action.yml

Reply:
Cool, thanks.